fec/fec/static/js/calendar.js

```diff
@@ -310,6 +313,12 @@ Calendar.prototype.handleRender = function(view) {
   } else if (this.$listToggles) {
     this.$listToggles.remove();
   }
+  this.$calendar.find('.fc-content').each(function(){
```

Comment:
Also, can we avoid hard-coding this ID here? How about setting `describedby` to something like `this.$calendar.attr('id')`?